### DISCUSSION TITLE
Add url login identifier on mobile for password manager autofill

### DISCUSSION THREAD

**Discussion by traktuner** [immich-app/immich#7924] [Feature Request]
### The bug

Hello,
I have my Immich account saved in my password manager with the corresponding url. 
When I open Immich and it’s not logged in, I first add the url and click next. This url should then be used as the „identifier“ for the username and password fields on the next page to make password manager autofill work, which would be a great quality-of-life improvement. 
I don’t know if that is already working with Android but it’s definitely not the case on iOS.

Thank you very much! 

### The OS that Immich Server is running on

Synology docker 

### Version of Immich Server

1.98.2

### Version of Immich Mobile App

1.98.2

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Not relevant
```

### Your .env content

```Shell
Mot relevant
```

### Reproduction steps

```bash
See my description above
```

**Comment by bvarick** [comment]
I'd also like to see this!
I think the relevant documentation for implementing this are here:

https://developer.apple.com/documentation/xcode/supporting-associated-domains

**Comment by mmomjian** [comment]
dup #3618
